boxes.yaml.example

```diff
@@ -44,6 +44,7 @@ centos7-bats-ci:
       bats_tests:
         - "swapfile.bats"
         - "fb-install-katello.bats"
```

Comment:
If the file is not used and not recommended, why should it stay? Archaeology lessons can still be done using the Git history, an unused file is just confusing. Same applies for `katello-bats` btw, I think it's unused by now?